.github/workflows/cypress.yml

```diff
@@ -79,3 +79,96 @@ jobs:
         with:
           name: cypress-screenshot-artifacts
           path: cypress/screenshots
+
+      - name: Archive Cypress Mochawesome test results
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: cypress-mochawesome-test-results-artifact-${{ matrix.ci_node_index }}
+          path: cypress/results
+
+  mochawesome-report:
+    if: ${{ always() }}
+    needs: [build]
+    name: Generate Mochawesome Report
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: testing-tools-team-dashboard-data
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"
+      
+      - name: Set BIGQUERY_SERVICE_CREDENTIALS Environment Variable
+        uses: "marvinpinto/action-inject-ssm-secrets@v1.1.1"
+        with:
+          ssm_parameter: "/dsva-vagov/testing-team/bigquery_service_credentials"
+          env_variable_name: "BIGQUERY_SERVICE_CREDENTIALS"
+
+      - name: Obtain GitHub Token
+        uses: marvinpinto/action-inject-ssm-secrets@latest
+        with:
+          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
+      
+      - name: Checkout Testing Tools Team Dashboard Data repo
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/testing-tools-team-dashboard-data
+          token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          path: testing-tools-team-dashboard-data
+      
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: vsp-analytics-and-insights
+          service_account_key: ${{ env.BIGQUERY_SERVICE_CREDENTIALS }}
+          export_default_credentials: true
+      
+      - name: Download test result artifact 0
+        uses: actions/download-artifact@v2
+        with:
+          name: cypress-mochawesome-test-results-artifact-0
+          path: testing-tools-team-dashboard-data/src/mochawesome/data
```

Comment:
I wonder if all the artifacts can be uploaded under a single folder so that you can just download the whole folder in one step. This is fine for now, though.

Reply:
That would be cool, but I'm not sure that's possible. I just reviewed [the Upload-Artifact v2documentation](actions/upload-artifact@v2). It seems you can upload files to the same artifact in different steps within the same job, but uploads to the same artifact from different jobs replace each other.

There's also this warning from those docs:
> Be careful when uploading to the same artifact via multiple jobs as artifacts may become corrupted